config/maintainers.json

```diff
@@ -113,7 +113,7 @@
     },
     {
       "github_username": "yyyc514",
```

Comment:
```suggestion
      "github_username": "joshgoebel",
```

@joshgoebel as I understand you've renamed your account.

Reply:
Correct. Thanks.